ci/docker/buildenv/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get install -y git build-essential  autoconf flex bison \
+                                         gcc-6 gcc-5 gcc-4.8 g++-6 g++-5 g++-4.8 \
+                                         bc perl python3 gdb
+
+RUN apt-get install -y wget
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/wsnyder/verilator/pullRequest?prid=4756540)